bitnami/keycloak/values.yaml

```diff
@@ -655,3 +655,78 @@ externalDatabase:
   ## Database name
   ##
   database: bitnami_keycloak
+
+##
+## Below is the implementation of Keycloak Configuration as Code.
+##
+KcCasC:
```

Comment:
What do you think about using a more descriptive name and following camel-case style? `keycloakConfigAsCode` maybe?